mdpow/equil.py

```diff
@@ -599,3 +647,8 @@ class OctanolSimulation(Simulation):
     solvent_default = 'octanol'
     dirname_default = os.path.join(Simulation.topdir_default, solvent_default)
 
+class WetOctanolSimulation(Simulation):
+    """Equilibrium MD of a solute in a box of wet octanol."""
+    solvent_default = 'wetoctanol'
+    dirname_default = os.path.join(Simulation.topdir_default, solvent_default)
+
```

Comment:
see comment above on keeping all specific WetOctanol code in the class for a more OO approach